specs/provisioningInfo.md

```diff
@@ -0,0 +1,94 @@
+
+## Notational Conventions
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 [RFC2119] [RFC8174] when, and only when, they appear in all capitals, as shown here.
+
+## <a name="permissionsObject"></a> Permissions JSON Object
+This is the top level object in the provisioning info file. It contains a list of permission claim values and the cloud environments in which they are present.
+
+```json
+"permissions": {
+  “Foo.Read.All”: {
+    "environments": {
+        "global": {
+          "versions": {
+            "v1": {
+                "schemes": [
+                  {
+                    "DelegatedWork": {
+                      "isPresent": true,
+                      "isHidden": false,
+                      "isDisabled": false,
+                      "id": "bed71653-0fa8-42a0-b33c-a1aab02ecda6"
+                    }
+                  }
+                ]
+            },
+            "beta": {
+                "schemes": [
+                  {
+                    "DelegatedWork": {
+                      "isPresent": true,
+                      "isHidden": false,
+                      "isDisabled": false,
+                      "id": "bed71753-0fq8-42a2-b33c-c1acb02ecd17"
+                    }
+                  }
+                ]
+            },
+          }
+        }
+    }
+  }
+}
+```
+
+In this example, the permission "Foo.Read.All" is present in the global environment in both v1 and beta versions. The "DelegatedWork" scheme is present in both versions. Note that the same scheme of the same permission have permission has different GUIDs in the two clouds.
+
+### permissions
+The "permissions" member is a JSON object whose members permission objects. The key of each member is the claim identifier used for the [Permission Object](#permissionObject)
+
+
+### <a name="permissionObject"></a>Permission Object
+This member is a JSON object that describes the provisioning info for a permission.
+
+#### environments
+The "environments" member is an array of objects that identifies the deployment environments in which the permission SHOULD be supported. The key of each member is the cloud environment identifier used for the [Cloud Environment Object](#cloudEnvironmentObject)
```

Comment:
Does this mean that we will have a block for each of the environments and versions where the permission is supported?